source/includes/_queries.md

```diff
@@ -40,13 +41,22 @@ Each query consists of following things:
     <li> **Query object** might be any of types, described below </li>
 </ul>
 
-**Signatures** contain one or many signatures (ed25519 pubkey + signature):
+**Signatures** contain one or many signatures (ed25519 pubkey + signature).
```

Comment:
I think we can remove note in the parentheses as long as hyperledger/iroha#795 almost merged

Reply:
Also, I do not think we use any other signature systems, so it is better to specify that keys are formed using ed25519, and then just say "keys". Moreover, link to ed25519 itself would be good for reference.